extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -170,6 +170,10 @@ export class PublishDatabaseDialog {
 								title: constants.selectConnectionRadioButtonsTitle,
 								component: selectConnectionRadioButtons
 							},*/
+							{
+								component: displayOptionsButton,
```

Comment:
please add screenshot of what this change looks like, before and after

Reply:
Actually this is just for representing the button as a new component instead of putting in the horizontal form section. This should go with CSS changes PR. will be adding there! as it required the CSS updates too!